antarest/study/web/study_data_blueprint.py

```diff
@@ -214,4 +214,15 @@ def edit_matrix(
         params = RequestParameters(user=current_user)
         study_service.update_matrix(uuid, path, slices, operation, params)
 
+    @bp.post(
```

Comment:
also add another endpoint (admin only) to update studies (create AdditionnalData) from files